arm-insights/2015-04-01/swagger/insightsClient.json

```diff
@@ -0,0 +1,1123 @@
+          },
+          {
+            "name": "api-version",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "The resource provider api version."
+          },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The odata filter string to use in the query"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful request to get the list of usage metric values",
+            "schema": {
+              "$ref": "#/definitions/UsageMetricCollection"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        },
+        "x-ms-odata": "#/definitions/UsageMetric"
+      }
+    }
+  },
+  "definitions": {
+    "LocalizableString": {
+      "properties": {
+        "value": {
+          "type": "string",
+          "description": "Gets or sets the invariant value."
+        },
+        "localizedValue": {
+          "type": "string",
+          "description": "Gets or sets the locale specific value."
+        }
+      },
+      "description": "The localizable string class."
+    },
+    "EventCategoryCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/LocalizableString"
+          },
+          "description": "Gets or sets this list that includes the Azure event categories."
+        }
+      },
+      "description": "Represents collection of event categories."
+    },
+    "SenderAuthorization": {
+      "properties": {
+        "action": {
+          "type": "string",
+          "description": "Gets or sets the Action."
+        },
+        "condition": {
+          "type": "string",
+          "description": "Gets or sets the condition."
+        },
+        "role": {
+          "type": "string",
+          "description": "Gets or sets the role."
+        },
+        "scope": {
+          "type": "string",
+          "description": "Gets or sets the scope."
+        }
+      },
+      "description": "The authorization class."
+    },
+    "HttpRequestInfo": {
+      "properties": {
+        "clientRequestId": {
+          "type": "string",
+          "description": "Gets or sets the client request id."
+        },
+        "clientIpAddress": {
+          "type": "string",
+          "description": "Gets or sets the client Ip Address"
+        },
+        "method": {
+          "type": "string",
+          "description": "Gets or sets the method."
+        },
+        "uri": {
+          "type": "string",
+          "description": "Gets or sets the Uri."
+        }
+      },
+      "description": "The HttpInfo class."
```

Comment:
We would want the specs to be in the best possible state. Because once the spec is merged it is consumed by a variety of applications for different purposes. Hence every aspect of it is super important. If time is indeed a problem then we can sit together and help you onboard faster.